webapp/app.py

```diff
@@ -63,6 +73,80 @@ def _get_examples():
     return examples
 
 
+def _make_github_request(endpoint):
+    # Passing a gh username as a header increases
+    # the API rate limit from 60 to 5000 per hour
+    headers = { "username": random.choice(team_members)["login"] }
+
+    try: 
+        response = requests.get(
+            f'https://api.github.com/{endpoint}',
+            headers,
+            timeout=1
```

Comment:
this timeout is really low, I don't think we have gone bellow `3` seconds anywhere